docs/artifacts.md

```diff
@@ -0,0 +1,175 @@
+<!--
+---
+linkTitle: "Artifacts"
+weight: 201
+---
+-->
+
+# Artifacts
+
+- [Overview](#overview)
+- [Artifact Provenance Data](#passing-artifacts-between-steps)
+  - [Passing Artifacts between Steps](#passing-artifacts-between-steps)
+
+
+
+## Overview
+> :seedling: **`Artifacts` is an [alpha](additional-configs.md#alpha-features) feature.**
+> The `enable-artifacts` feature flag must be set to `"true"` to read or write artifacts in a step.
+
+Artifacts provide a way to track the origin of data produced and consumed within your Tekton Tasks.
+
+## Artifact Provenance Data
+Artifacts fall into two categories:
+
+ - Inputs: Artifacts downloaded and used by a Task.
```

Comment:
```suggestion
 - Inputs: Artifacts downloaded and used by a Task.
 Inputs: Artifacts downloaded by the `Step/Task`.
```
It can be used by another Step/Task. e.g. git clone to a shared workspace is used by multiple Steps/Tasks.